RHEL/6/input/remediations/bash/templates/Makefile

```diff
@@ -24,3 +24,7 @@ copy:
 clean:
 	rm -f output/*.sh
 	rm -rf output/
+
+clean_copied:
```

Comment:
@ybznek I wonder if the build system should get the templates from `templates/output` instead of this target?